truewiki/web_routes.py

```diff
@@ -192,6 +192,14 @@ async def edit_page_post(request):
     if "content" not in payload:
         raise web.HTTPNotFound()
     content = payload["content"].replace("\r", "")
+    summary = payload["summary"].replace("\r", "")
+
+    # Make sure summary is not more than 500 chars. This is normally limited
+    # by the HTML form itself, so if this is somehow longer, people have
+    # been messing with the HTML. We should probably just reject the request
+    # at this point.
+    if len(summary) > 500:
+        return web.HTTPBadRequest(reason="Summary exceeds maximum length of 500 characters.")
```

Comment:
aiohttp prefers if this is a `raise`. It still works for their current release, but I believe they want to remove that support. so:

`raise web.HTTPBadRequest`